NEWS.md

```diff
@@ -11,6 +11,13 @@
   `mmrm(..., start = start, optimizer = c("BFGS", "nlminb"))`. These arguments will go into `mmrm_control`.
   In addition, `optimizer = "automatic"` is deprecated and will give you warning. It is a default behavior now to
   try all optimizers (if multiple optimizer provided) if the first optimizer fails.
+- Add new arguments to `mmrm_control` to allow users to keep all levels in visits. This allows user to specify the distance of
```

Comment:
```suggestion
- Add new argument `drop_visit_levels` to allow users to keep all levels in visits. This allows user to specify the distance of
```

Reply:
I would move the details what this allows to do etc. to the `mmrm_control ` function documentation, I feel a bit too much detail for the `NEWS` here

Reply:
here in addition please mention that now a warning is raised if a visit level is dropped, which was done silently before

Reply:
This is still too much detail , please move to `mmrm_control` documentation